app/assets/stylesheets/_navbar.sass.scss

```diff
@@ -0,0 +1,39 @@
+.navbar-collapse {
+  justify-content: space-evenly;
+  padding-left: 13rem;
+}
+.nav-item {
+  margin: 0 2.5rem 0 2.5rem;
+}
+
+.navbar {
+  padding-left: 1rem;
+  padding: 0.9rem;
+  background: $dark-blue;
+}
+
+.navbar-light .navbar-nav .nav-link {
+  color: $navbar-text;
+  font-weight: 600;
+}
+
+.navbar-light .navbar-nav .nav-link:hover {
+  color: $white-color;
+}
+
+
+.navbar-light .navbar-brand {
+  color: $white-color;
+  margin-left: 5rem;
+  font-weight: 800;
+  font-size: 1.5rem;
+}
+
+.navbar-light .navbar-brand:hover {
+  color: $white-color;
+}
```

Comment:
You can group them, since they all derive from .navbar-light

Reply:
@YilKanda thanks, that's it